src/main/resources/static/index.html

```diff
@@ -13,6 +13,10 @@ <h1>Coach App</h1>
   <input type="number" id="weight" placeholder="weight">
   <input type="number" id="sets" placeholder="sets">
   <input type="number" id="reps" placeholder="reps">
+  <div id="full-program"></div>
+  <div id="day-exercise">  
```

Comment:
I would move this above the inputs, so as new exercises are added the input stays on the bottom of the list